Index: src/com/team2/eduops/controller/CheckInOut.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/com/team2/eduops/controller/CheckInOut.java b/src/com/team2/eduops/controller/CheckInOut.java
new file mode 100644
--- /dev/null	(date 1721635434941)
+++ b/src/com/team2/eduops/controller/CheckInOut.java	(date 1721635434941)
@@ -0,0 +1,39 @@
+package com.team2.eduops.controller;
+
+import java.sql.*;
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+
+
+public static void class CheckInOut {
+    Connection conn = DriverManager.getConnection(url,user,password);
+    PreparedStatement pstmt = conn.
+        //입실
+        public static void checkIn(std_no) throws SQLException,ClassNotFoundException {
+            System.out.println("입실이 완료되었습니다.");
+            LocalDate today = LocalDate.now();
+            Timestamp currentTimestamp = new Timestamp(System.currentTimeMillis());
+
+            DateTimeFormatter formattedtime = DateTimeFormatter.ofPattern("HH:mm:ss");
+
+
+            pstmt = conn. prepareStatement("INSERT INTO ATTENDANCE VALUES( ?,?,?)");
+            pstmt.setDate(2, today);
+            pstmt.setInt(3, std_no);
+            //update
+            pstmt = conn.prepareStatement("UPDATE ATTENDANCE set CI_TIME = "+ currentTimestamp + " where STD_NO = "+ std_no +" and ATTEND_DATE = "+today);
+    }
+
+    //퇴실
+    public static void checkOut(std_no) throws SQLException,ClassNotFoundException{
+        System.out.pirntln("퇴실이 완료되었습니다.");
+        LocalDate today = LocalDate.now();
+        Timestamp currentTimestamp = new Timestamp(System.currentTimeMillis());
+
+        DateTimeFormatter formattedtime = DateTimeFormatter.ofPattern("HH:mm:ss");
+
+        pstmt = conn.prepareStatement("UPDATE ATTENDANCE set CO_TIME = "+currentTimestamp+ " where STD_NO = "+std_no+"and ATTEND_DATE = "+today);
+
+    }
+}
